notebooks/0_intro.ipynb

```diff
@@ -6,11 +6,11 @@
     "tags": []
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    nwb_file_name2 = filename + '_' + file_extension
</pre><p>Can we call nwb_file_name2, nwb_copy_file_name?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LorenFrankLab/spyglass/pull/314/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>